docs_src/microservices/core/metadata/details/DeviceProfile.md

```diff
@@ -1,12 +1,16 @@
-# Device Profile
+---
+title: Core Metadata - Device Profile
+---
+
+# Core Metadata - Device Profile
 
 The device profile describes a type of [device](../../../general/Definitions.md#device) within the EdgeX system. Each
 device managed by a device service has an association with a device profile,
 which defines that device type in terms of the operations which it supports.
 
-For a full list of device profile fields and their required values see the [device profile reference](./Ch-DeviceProfileRef.md).
+For a full list of device profile fields and their required values see the [device profile reference](./DeviceProfileRef.md).
```

Comment:
Can we just move the contents DeviceProfileRef.md to a subsection at the bottom of this file and remove DeviceProfileRef.md? Less files and and less navigating.

Reply:
fixed

Reply:
Have you pushed this change? I am not seeing it...